src/templates/bespoke/bespoke.scss

```
@@ -194,11 +194,14 @@ $progress-height: 5px;
 
       // Grid layout for presenter view
       display: grid;
+
+      --bespoke-marp-presenter-split-ratio: 66%;
+
       grid-template:
-        'current next' minmax(140px, 1fr)
-        'current note' 2fr
-        'info    note' 3em /
-        2fr 1fr;
+        'current dragbar next' minmax(140px, 1fr)
+        'current dragbar note' 2fr
+        'info    dragbar note' 3em;
+      grid-template-columns: var(--bespoke-marp-presenter-split-ratio) 0 1fr;
```

For saving a few bytes in output HTML:

```suggestion
        'current dragbar next' minmax(140px, 1fr)
        'current dragbar note' 2fr
        'info    dragbar note' 3em /
        var(--bespoke-marp-presenter-split-ratio) 0 1fr;
```

---

i tried that but the linter didn't like it, thoughts?

---

The documentation says [`scss/operator-no-newline-after`](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-newline-after/README.md) rule should ignore `/` used as a delimiter for CSS. I think it may be false positive :(

It's no problems to keep `grid-template-column`.